src/XMLimport.cpp

```diff
@@ -1878,10 +1878,13 @@ void XMLimport::readProfileShortcuts() {
         if (isStartElement()) {
             if (name() == "profileShortcut") {
                 auto key = attributes().value(QStringLiteral("key"));
-                auto sequenceString = readElementText();
-                QKeySequence* sequence = !sequenceString.isEmpty() ? new QKeySequence(sequenceString) : new QKeySequence();
-                mpHost->profileShortcuts.value(key.toString())->swap(*sequence);
-                delete sequence;
+                if (mpHost->profileShortcuts.value(key.toString())) {
+                    auto sequenceString = readElementText();
```

Comment:
BugFix: you must not skip the `readElementText()` even if the `if()` is not true - otherwise it doesn't advance the current position in the `QXmlStreamReader` and that breaks things massively thereafter in the file:
```suggestion
                auto sequenceString = readElementText();
                if (mpHost->profileShortcuts.value(key.toString())) {
```

Reply:
Fixed